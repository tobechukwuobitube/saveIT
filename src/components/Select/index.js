import React from 'react';

const accounts = [
  'Savings',
  'Current',
];

const SelectAccountType = ({ handleChange, type, selectedAccountType }) => (
  <select
    name={type}
    value={selectedAccountType}
    onChange={handleChange}
  >
    {accounts.map(account => {
      return (
        <option key={account} value={account}>
          {account}
        </option>
      );
    })}
  </select>
);

export default SelectAccountType;
