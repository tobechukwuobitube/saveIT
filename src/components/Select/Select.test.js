import React from 'react';
import renderer from 'react-test-renderer';
import Select from './index';

test('Select snapshot test', () => {
  const wrapper = renderer.create(<Select />);
  expect(wrapper).toMatchSnapshot();
});
