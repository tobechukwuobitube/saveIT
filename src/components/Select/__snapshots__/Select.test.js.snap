// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select snapshot test 1`] = `
<select>
  <option
    value="Savings"
  >
    Savings
  </option>
  <option
    value="Current"
  >
    Current
  </option>
</select>
`;
